File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/PromotionService.java
package com.pinyougou.sellergoods.service;

import java.util.List;

import com.pinyougou.pojo.TbBrand;
import com.pinyougou.pojo.TbPromotion;

import entity.PageResult;

public interface PromotionService {

	List<TbPromotion> findAll();


	void add(TbPromotion promotion);


	PageResult findPage(int pageNo, int pageSize);


	PageResult findPage(int pageNo, int pageSize, TbPromotion promotion);


	void delete(Long[] ids);


	TbPromotion findOne(Long id);


	void update(TbPromotion promotion);


}

File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/promotionService.js
app.service('promotionService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../promotion/findAll');		
	}
	
	this.search=function(pageNo,pageSize,searchEntity){
	 	return $http.post("../promotion/search/"+pageNo+"/"+pageSize,searchEntity);
	}
	
	 this.findPage=function(pageNo,pageSize){
		 return $http.get("../promotion/findPage/"+pageNo+"/"+pageSize);
	 }
	 
	//增加 
	this.add=function(entity){
		
		return  $http.post('../promotion/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../promotion/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../promotion/delete/'+ids);
	}
	
	this.findOne=function(id){
		return $http.get('../promotion/findOne/'+id);
	}

	
	
	
	
});
